components/03-modules/minicart-product/_minicart-product.scss

```diff
@@ -0,0 +1,127 @@
+
+$minicart-product__action-width              : 50% !default;
+$minicart-product__action-button-bg          : transparent !default;
+$minicart-product__action-icon-color         : $color-secondary !default;
+$minicart-product__action-icon-color--active : $color-primary !default;
+
+.minicart-product {
+    display: flex;
+    width: 100%;
+    flex-flow: row nowrap;
+    padding: $minicart-product__padding;
+    border-bottom: $minicart-product__border;
+
+    &:last-child {
+        border-bottom: none;
+    }
+
+    &__image {
+        width: $minicart-product__image-width;
+        img {
+            width: $minicart-product__image-width;
+            max-width: $minicart-product__image-width;
+        }
+    }
+
+    &__info {
+        display: flex;
+        flex-flow: row wrap;
+        width: calc(100% - $minicart-product__image-width);
+        padding: $minicart-product__info-padding;
+    }
+
+    &__name {
+        margin-bottom: $minicart-product__spacer;
+        font-weight: $font-weight-normal;
+    }
+
+    &__attributes {
+        margin-bottom: $minicart-product__spacer;
+
+        &-button {
+            padding: 0;
+            color: $minicart-product__attributes-button-color;
+            background-color: $minicart-product__attributes-button-bg-color;
+
+            &:focus,
+            &:hover {
+                background-color: $minicart-product__attributes-button-bg-color;
+            }
+
+            .dropdown-list__icon {
+                position: static;
+                margin-left: $spacer;
+                vertical-align: top;
+                fill: $minicart-product__attributes-button-color;
+            }
+        }
+
+        &-list {
+            padding: $minicart-product__attributes-list-padding;
+            color: $minicart-product__attributes-list-color;
+        }
+    }
+
+    &__price {
+        width: 100%;
+        font-weight: $font-weight-bold;
+    }
+
+    &__bottom {
+        display: flex;
+        justify-content: space-between;
+        width: 100%;
+    }
+
+    &__qty {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        flex: 1 $minicart-product__qty-width;
+        max-width: $minicart-product__qty-width;
+    }
+
+    &__actions {
+        display: flex;
+        justify-content: flex-end;
+        width: $minicart-product__action-width;
+    }
+
+    &__button {
+        margin-left: $spacer;
+        padding: 12px;
+        border: 0;
+        background-color: $minicart-product__action-button-bg;
+
+        &-icon {
+            height: 24px;
+            width: 24px;
+            padding: 2px;
+            fill: $minicart-product__action-icon-color;
+            &:focus,
+            &:hover {
+                fill: $minicart-product__action-icon-color--active;
+            }
+        }
+
+        &:focus,
+        &:hover {
+            background-color: $minicart-product__action-button-bg;
+            border: 0;
```

Comment:
there is no effect on hover fot those buttons?

Reply:
sure, removed.